backoffice-manage/src/main/resources/application.yml

```
@@ -32,3 +36,50 @@ springdoc:
+
+# Actuator 설정 추가
+management:
+  endpoints:
+    web:
+      exposure:
+        include: health,info
+  endpoint:
+    health:
+      show-details: always
+# Actuator 설정 추가
+management:
```

YAMLlint (1.37.1)

[error] duplication of key "management" in mapping

(key-duplicates)